src/packs/abilities/Talent_W4G9rOoBWQqffx2T/Level_8_qNF3sTx3kg7GPouS/11_Clarity_2Bsc4W8gkDQbJTOg/ability_Doubt_07fuK6yFfpbC2fAm.json

```
@@ -0,0 +1,184 @@
+      "page": "199",
+      "license": "Draw Steel Creator License"
+    },
+    "_dsid": "doubt",
+    "story": "You tug at the strings of the foe’s anima and unravel them, allowing someone else to take advantage of their drive.",
+    "keywords": [
+      "psionic",
+      "ranged",
+      "strike",
+      "animapathy"
+    ],
+    "type": "main",
+    "category": "heroic",
+    "resource": 11,
+    "trigger": "",
+    "distance": {
+      "type": "ranged",
+      "primary": 10
+    },
+    "damageDisplay": "melee",
+    "target": {
+      "type": "creatureObject",
+      "custom": "",
+      "value": 1
+    },
+    "power": {
+      "roll": {
+        "formula": "@chr",
+        "characteristics": [
+          "presence"
+        ]
+      },
+      "effects": {
+        "QR4KPgHr947z7Sdf": {
+          "name": "",
+          "img": null,
+          "type": "damage",
+          "_id": "QR4KPgHr947z7Sdf",
+          "damage": {
+            "tier1": {
+              "value": "10 + @chr",
+              "potency": {
+                "characteristic": "none"
+              }
+            },
+            "tier2": {
+              "value": "14 + @chr"
+            },
+            "tier3": {
+              "value": "20 + @chr"
+            }
+          }
+        },
+        "MySdHpY6mNAVyGqR": {
+          "name": "",
+          "img": null,
+          "type": "applied",
+          "_id": "MySdHpY6mNAVyGqR",
+          "applied": {
+            "tier1": {
+              "potency": {
+                "characteristic": "presence"
+              },
+              "display": "{{potency}}, weakened (save ends)",
+              "effects": {
+                "slowed": {
+                  "condition": "failure",
+                  "end": "save"
+                }
+              }
+            },
+            "tier2": {
+              "display": "{{potency}}, weakened (save ends)",
+              "effects": {
+                "slowed": {
+                  "condition": "failure",
+                  "end": "save"
+                }
+              }
+            },
+            "tier3": {
+              "display": "{{potency}}, weakened and slowed (save ends)",
+              "effects": {
+                "weakened": {
+                  "condition": "failure",
+                  "end": "save"
+                },
+                "slowed": {
+                  "condition": "failure",
+                  "end": "save"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "effect": {
+      "before": "<p>This ability gains an edge against a target with a soul. After you make the power roll, you or one ally within distance have a double edge on the next power roll you make before the end of the encounter.</p><p><strong>Strained:</strong> You feel dispirited until you finish a respite. If you obtain a tier 3 outcome on the power roll, you and the target each have [[/apply \"Damage Weakness 5\"]] (save ends).</p>",
```

```suggestion
      "before": "<p>This ability gains an edge against a target with a soul. After you make the power roll, you or one ally within distance have a double edge on the next power roll you make before the end of the encounter.</p><p><strong>Strained:</strong> You feel dispirited until you finish a respite. If you obtain a tier 3 outcome on the power roll, you and the target each have [[/apply atsPk0jYryV6sL3q]] (save ends).</p>",
```

Prefer Ids to name matching